src/Aliases.lua

```diff
@@ -1,2 +1,6 @@
 set = setsetting
 get = getsetting
+
+getopt = getuseroption
+useropt = getuseroption
```

Comment:
I don't really think `useropt` would be a good alias. It's not intuitive (is it setting or getting an option?) and is a name I see a lot of people accidentally overriding. I really like `getopt` and `setopt` though. Don't you think it would be better if we kept only the 2 latter?